cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -556,13 +564,30 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
         return result
     }
 
-    internal fun handleArgument(node: Python.ASTarg, isPosOnly: Boolean = false) {
+    internal fun handleArgument(
```

Comment:
doc string

Reply:
Done

Reply:
I fail to see it. Is this some Java / Intellij magic again?

Reply:
Sorry, my bad. 
Done.